### DISCUSSION TITLE
Feature request: change font in viewer to be able to show text in fixsize fonts.

### DISCUSSION THREAD

**Discussion by cstenkes** [mucommander/mucommander#1081] [Ideas]
Feature request: change font in viewer to be able to show text in fixsize fonts.

**Comment by pskowronek** [comment]
![obraz](https://github.com/mucommander/mucommander/assets/360665/90152b03-896c-4b3d-8781-19184e01ba8e)

What about this?

**Comment by cstenkes** [comment]
Hi, 
I haven't find this fuctionality at first sight ;)
I tested and it is working fine.

BR, Csaba
